### DISCUSSION TITLE
How to use esc key to quit history search

### DISCUSSION THREAD

**Discussion by yaozw** [marlonrichert/zsh-autocomplete#599] [Q&A]
Hi All,

I am new to this great plugin, currently I have to use ctrl+c to quit history search, I want to change to esc key, can someone tell me how to do? Thanks a lot!

**Comment by marlonrichert** [comment]
Replacing Esc with Ctrl-C is not easily possibly in a normal terminal + shell. I suggest you ask about this on https://unix.stackexchange.com/
